src/app/modules/core/manager/reservation/components/list/manager-reservation-list.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
```

Comment:
Please, rename the file, the component class, all the related files and the selector in `reservation-list` instead of `manager-reservation-list`. Most probably this component will be shared with the landlord module. Thanks.